src/jvmMain/kotlin/io/github/fourlastor/wilds_launcher/extensions/InputStreamCaptureExtension.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.fourlastor.wilds_launcher.extensions
+
+import java.io.InputStream
+import java.io.InputStreamReader
+
+fun InputStream.capture(log: (String) -> Unit) {
```

Comment:
Making this [inline](https://kotlinlang.org/docs/inline-functions.html) will avoid allocating an object every time it's called (as a rule of thumb, if you accept a lambda, chances are you want it to be inline)
```suggestion
inline fun InputStream.capture(log: (String) -> Unit) {
```

Reply:
Oh cool, didn't even know about the inline syntax!